src/output-json-email-common.c

```diff
@@ -295,17 +296,31 @@ static bool EveEmailLogJsonData(const Flow *f, void *state, void *vtx, uint64_t
         int url_cnt = 0;
         JsonBuilder *js_attch = jb_new_array();
         JsonBuilder *js_url = jb_new_array();
+        bool has_ipv6_url = false;
+        bool has_ipv4_url = false;
+        bool has_exe_url = false;
         if (entity->url_list != NULL) {
             MimeDecUrl *url;
             for (url = entity->url_list; url != NULL; url = url->next) {
                 char *s = BytesToString((uint8_t *)url->url,
                                         (size_t)url->url_len);
                 if (s != NULL) {
                     jb_append_string(js_url, s);
+                    if (url->url_flags) {
```

Comment:
formatting is off

Also, can it just be 3 lines like
`has_exe_url = ((url->url_flags & URL_IS_EXE) != 0);`